File: C_Java_Exercises/pattern1.c
#include<stdio.h>
int main(){
    int n;
    printf("Enter no:");
    scanf("%d",&n);
    int x='A';
    for(int i=1;i<=n;i++){
        for(int j=1;j<=i;j++){
            printf("%c",(char)x);
           
        }
        x++;
        printf("\n");
        int y='B';
        printf("%c",(char)x);
    }
    return 0;
}

File: C_Java_Exercises/pattern1.java
import java.util.Scanner;

public class pattern1 {
    public static void main(String args[]){
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter no ");
        int n=sc.nextInt();
        int a=n;
        for(int i=1;i<=n;i++){

            for(int j=1;j<a;j++){
                System.out.print(a);

            }
            
            System.out.println(a);
         a--;
        }
       
      }
}
